README.md

```diff
@@ -209,6 +209,18 @@ final ApnsClient apnsClient = new ApnsClientBuilder()
     .build();
 ```
 
+If using HTTP proxies configured via JVM system properties, you can also use:
```

Comment:
As a minor thing, it might be helpful to link to the docs for the system properties.

```suggestion
If using HTTP proxies configured via [JVM system properties](https://docs.oracle.com/javase/7/docs/api/java/net/doc-files/net-properties.html#Proxies), you can also use:
```